src/components/Board/Output/Output.container.js

```diff
@@ -179,6 +181,14 @@ export class OutputContainer extends Component {
     this.clearOutput();
   };
 
+  handleCopyClick = () => {
+    const { intl, showNotification } = this.props;
+    let labels = [];
```

Comment:
I know this was merged already, but I have a very small comment. 

It would be simpler to assign the value when declaring the variable, and it's usually preferable to use `forEach` over `map` when we don't care about the function's return value.

```javascript
const labels = this.props.output.map(symbol => symbol.label);
```

Reply:
That makes sense. Just submitted a request for this update, thanks!